examples/stable_diffusion_xl/tools/model_conversion/README.md

```diff
@@ -0,0 +1,97 @@
+# Model Conversion between Mindspore and Torch Inference
+
+We provide scripts of the checkpoint conversion between `.safetensors` and `.ckpt` format. For training or inference in Mindspore, we use the pre-trained weights from hugging face and convert them to `.ckpt` format. Please refer to [the preparation part of GETTING_STARTED.md](https://github.com/mindspore-lab/mindone/blob/master/examples/stable_diffusion_xl/GETTING_STARTED.md#convert-pretrained-checkpoint) for details. After finetuning in Mindspore, we can convert the checkpoints back for torch inference as well.
+
+The tutorial shows how to run inference in the official SDXL repo, [generative-models](https://github.com/Stability-AI/generative-models), with Mindspore checkpoints. Please refer to this [tutorial](https://github.com/mindspore-lab/mindone/blob/master/examples/stable_diffusion_xl/tools/lora_conversion/README.md) if running in diffusers.
+
+## Getting started
+
+Notes: if you use Vallina fine-tune, or set `lora_merge_weights` to `true` when using LoRA, you directly get the finetuned weight of the whole model, please skip step 1.
+
+**Step 1. Merge fine-tuned LoRA weights to the pertained base weight**
+
+According to LoRA, $W_{finetuned} = W_{pretrained}+B\times A\times scale$. We can merge our trained LoRA weights with the pre-trained weights by running,
+
+```shell
+python merge_lora_to_base.py \
+	--weight_lora {path to mindspore lora ckpt} \
+	--weight_pretrained ./checkpoints/sd_xl_base_1.0_ms.ckpt \
+	--weight_merged ./checkpoints/sd_xl_base_finetuned_ms.ckpt
+```
+
+The default path of the merged checkpoint is `./checkpoints/sd_xl_base_finetuned_ms.ckpt`.
+
+**Step 2. Convert ms checkpoint to pt**
+
+To convert the finetuned mindspore checkpoints, run as follows.
+
+```shell
+python convert_weight.py \
+	--task ms_to_pt
+  --weight_safetensors ./checkpoints/sd_xl_base_finetuned_pt.safetensors \
+  --weight_ms ./checkpoints/sd_xl_base_finetuned_ms.ckpt  \
+  --key_torch torch_key_base.yaml \
+  --key_ms mindspore_key_base.yaml
+```
+
+The default path of the converted checkpoint is `./checkpoints/sd_xl_base_finetuned_pt.safetensors`.
+
+**Step 3. Run inference in [generative-models](https://github.com/Stability-AI/generative-models)**
+
+Replace the ckpt path of SDXL-base-1.0 with `sd_xl_base_finetuned_pt.safetensors` at the constant `VERSION2SPECS` in `scripts/demo/sampling.py`, as well as the prompt in ``__main__`.  Then you run inference in generative-models repo with our fine-tuned checkpoint.
+
+```shell
+# run in generative-models repo
+streamlit run scripts/demo/sampling.py --server.port <your_port>
+```
+
+**Step 4. Check consistency between PT and MS inference results (optional)**.
```

Comment:
this section is duplicated with https://github.com/mindspore-lab/mindone/tree/master/examples/stable_diffusion_xl/tools/lora_conversion#3-check-consistency-between-pt-and-ms-inference-results-optional

It's better to link to it in case for future code or doc change